modules/varnish/templates/default.vcl

```diff
@@ -529,6 +529,11 @@ sub vcl_recv {
 		}
 	}
 
+        if (req.http.Host ~ "^*.betaheze.org") {

```

Comment:
This is invalid regex

Reply:
This should be `^(.*\.)?betaheze\.org` in order to also do betaheze.org itself, or `^.*\.betaheze\.org` for just subdomains. I'm not certain if the periods need escaped or not though.

Reply:
> I'm not certain if the periods need escaped or not though.

They don't *strictly* need to be, but should be.